tests/conftest.py

```diff
@@ -1,32 +1,99 @@
 # coding=utf-8
+# pylint: disable=redefined-outer-name
+from datetime import datetime, timedelta
+from unittest.mock import patch
+from uuid import uuid4
+
 import pytest
 from multidict import CIMultiDict
 
+from lbz.authentication import User
+from lbz.authz.authorizer import Authorizer
 from lbz.dev.misc import Event
 from lbz.request import Request
+from tests import SAMPLE_PRIVATE_KEY
+from tests.fixtures.rsa_pair import SAMPLE_PUBLIC_KEY
+from tests.utils import encode_token, allowed_audiences
 
 
-@pytest.fixture
-def sample_request():
+@pytest.fixture()
+def sample_request() -> Request:
     return Request(
-        headers=CIMultiDict({"Content-Type": "application/json"}),
-        uri_params={},
         method="GET",
         body="",
+        headers=CIMultiDict({"Content-Type": "application/json"}),
+        user=None,
+        uri_params={},
         context={},
+        query_params=None,
         stage_vars={},
         is_base64_encoded=False,
-        query_params=None,
-        user=None,
     )
 
 
-@pytest.fixture
-def sample_event():
+@pytest.fixture()
+def sample_event(sample_request) -> Event:
     return Event(
         resource_path="/",
         method="GET",
         headers={},
         path_params={},
         query_params={},
+        body=sample_request.to_dict(),
+    )
+
+
+def _base_auth_payload() -> dict:
```

Comment:
Shouldn't it be a `full_access_authz_payload`? You are granting access to everything here, so it shouldn't be simple base, I think 😉